examples/flat-serialisation/anonymous-owner.json

```diff
@@ -0,0 +1,50 @@
+{
+  "version": "0.1",
+  "statementGroups": [
+    {
+      "statementID": "32e09b90-203d-42d1-bd77-a6465bca2552",
+      "beneficialOwnershipStatements": [
+        {
+          "statementID": "0e5c86db-f2b0-4e11-bff0-db9eee3db221",
+          "statementDate": "2017-07-18",
+          "subject": {
+            "subjectEntity": {
+              "type": "entityStatement",
+              "statementID": "77ee64ba-858e-4413-bfaf-991be2200f71"
```

Comment:
I've been wondering about whether using 'describedByStatement' as the property name here can help communicate the idea that we're linking to a description of the entity / person, rather than to the identified thing itself?

Reply:
Yes, good idea.